tasks/DrawPrize.js

```diff
@@ -0,0 +1,61 @@
+const { ethers } = require('ethers');
+const { cyan } = require('chalk');
+const { range } = require('./utils/range');
+const { getUserAndWallet } = require('./utils/getUserAndWallet');
+const { prepareClaims, batchCalculateDrawResults } = require('@pooltogether/draw-calculator-js');
+
+/**
+ * @name DrawPrize.claim()
+ */
+ task("claim", "Claim prizes from DrawPrizs")
+ .addOptionalParam("user", "<address>")
+ .addOptionalParam("wallet", "<address>")
+ .setAction(async (args, {ethers}) => {
+    const { user, wallet } = await getUserAndWallet(ethers, args)
+    const drawPrize = await (await ethers.getContract('DrawPrize')).connect(wallet)
+    const drawHistory = await ethers.getContract('DrawHistory')
+    const drawCalculatorContract = await ethers.getContract('DrawCalculator')
+    const prizeDistributionHistory = await ethers.getContract('PrizeDistributionHistory')
+    
+    // READ Draw Range
+    const newDraw = await drawHistory.getNewestDraw()
+    const oldDraw = await drawHistory.getOldestDraw()
+    const list = range((newDraw.drawId - oldDraw.drawId), oldDraw.drawId) // Generate Draw.drawId list [1,2,4,5,6,7]
+    
+    // READ PrizeDistribution list
+    const drawList = await drawHistory.getDraws(list)
+    const prizeDistributionList = (await prizeDistributionHistory.getPrizeDistributions(list))
```

Comment:
Could you use the `getDrawAndPrizeDistributionHistory` helper here to help you retrieve these values?